build.rs

```diff
@@ -17,10 +17,11 @@ fn main() {
         config.generator(s);
     }
     let dst = config
-        .define("WITH_BZip2", "OFF")
-        .define("WITH_HarfBuzz", "OFF")
-        .define("WITH_PNG", "OFF")
-        .define("WITH_ZLIB", "OFF")
+        .define("CMAKE_DISABLE_FIND_PACKAGE_BZip2", "TRUE")
+        .define("CMAKE_DISABLE_FIND_PACKAGE_HarfBuzz", "TRUE")
+        .define("CMAKE_DISABLE_FIND_PACKAGE_PNG", "TRUE")
+        .define("CMAKE_DISABLE_FIND_PACKAGE_ZLIB", "TRUE")
+        .define("CMAKE_DISABLE_FIND_PACKAGE_BrotliDec", "TRUE")
```

Comment:
This is the only manual change which was necessary and it's an interesting one.

Apparently FreeType changed up their build system, changing configuration flags like `WITH_Bzip2` to `FT_WITH_BZIP2`. Important here is the capitalization got changed, so it's now all caps.

However, to completely disable the loading of the libraries through cmake, the additional variable `CMAKE_DISABLE_FIND_PACKAGE_BZip2` was introduced. These use the same capitalization as the old variables.

So it's probably helpful not to try `FT_WITH_BZIP2` first and then just replace the `FT_WITH_` part, because it took me way too long to figure out that my capitalization was wrong :'). Hopefully that should explain this change and maybe help others trying to do the same.